exemplar/origin/origin.go

```diff
@@ -0,0 +1,55 @@
+package origin
+
+import (
+	"context"
+	"time"
+
+	"ftl/builtin"
+
+	"github.com/TBD54566975/ftl/go-runtime/ftl" // Import the FTL SDK.
+)
+
+var nonce = ftl.Config[string]("nonce")
+
+//ftl:export
+var AgentBroadcast = ftl.Topic[Agent]("agentBroadcast")
+
+type Agent struct {
+	ID            int                   `json:"id"`
+	Alias         string                `json:"alias"`
+	LicenseToKill bool                  `json:"license_to_kill"`
+	HiredAt       time.Time             `json:"hired_at"`
+	BriefedAt     ftl.Option[time.Time] `json:"briefed_at"`
+}
+
+type PostAgentResponse struct {
+	ID int `json:"id"`
+}
+
+type PostAgentErrorResponse string
+
+//ftl:ingress POST /http/agent
+func PostAgent(ctx context.Context, req builtin.HttpRequest[Agent, ftl.Unit, ftl.Unit]) (builtin.HttpResponse[PostAgentResponse, PostAgentErrorResponse], error) {
+	agent := Agent{
+		ID:            req.Body.ID,
+		Alias:         req.Body.Alias,
+		LicenseToKill: req.Body.LicenseToKill,
+		HiredAt:       req.Body.HiredAt,
+	}
+	AgentBroadcast.Publish(ctx, agent)
+	return builtin.HttpResponse[PostAgentResponse, PostAgentErrorResponse]{
+		Status:  201,
+		Headers: map[string][]string{"Post": {"Header from FTL"}},
```

Comment:
This header is optional, unless you're testing against it's value you can probably safely remove.